EventsExpress/Controllers/AuthenticationController.cs

```diff
@@ -135,11 +146,11 @@ public async Task<IActionResult> GoogleLogin([FromBody]UserView userView)
         public IActionResult Login()
         {
             var user = _authService.GetCurrentUser(HttpContext.User);
-            var userInfo = _mapper.Map<UserInfo>(user);
-
-            return Ok(userInfo);
+            return
+            user == null
+               ? (IActionResult)Unauthorized()
```

Comment:
Is this cast really necessary? Please remove it if not.

Reply:
Yes its necessary ? because there is no implicit conversion between '...BadRequestObjectResult' and '...OkResult'.